main-settings/src/main/scala/sbt/std/InputWrapper.scala

```diff
@@ -182,12 +182,12 @@ object ParserInput {
       }
     }
 
-  private def wrapInputTask[T](c: Context)(tree: c.Tree, pos: c.Position) = {
+  private def wrapInputTask[T: c.WeakTypeTag](c: Context)(tree: c.Tree, pos: c.Position) = {
```

Comment:
So we're thread safe because 2.11? - http://docs.scala-lang.org/overviews/reflection/thread-safety.html